.github/workflows/dispatch-update-to-build-ci.yml

```diff
@@ -0,0 +1,48 @@
+name: "Dispatch Event to Build CI"
+on:
+  push:
+    branches:
+      - main
+    tags: 
+      - '*.*.*'
+    paths:
+      - '**.py'
```

Comment:
Possibly need to check if files other than.py are changed?

For example packages/cice5/spack-build.sh

Reply:
Noted and changed :)

Reply:
Could also change to
```yaml
    paths:
      - 'packages/**'
```
to avoid triggering if README or `repo.yaml` is changed.

Reply:
Good point!! Will add now